docs/en_US/awesome/tools.md

```diff
@@ -6,42 +6,75 @@
 
 <Uuid />
 
-There is a third-party UUID generator [uuidgenerator.net](https://www.uuidgenerator.net).
-
+There's also a third-party UUID generator [uuidgenerator.net](https://www.uuidgenerator.net)
 
 ## Third-party GUI Clients
 
+<Tool
+    url="https://github.com/2dust/v2rayN"
+    name="V2rayN"
+    :platforms="['win']"
+    description="[Actively Updated] V2RayN is a Windows client based on V2Ray, Singbox, and other cores."
+/>
 
 <Tool
-    url="https://github.com/2dust/v2rayNG"
-    name="V2RayNG"
-    :platforms="['android']"
-    description="V2RayNG is an Android APP based on V2Ray core, VPN connections can be created with VMess."
+    url="https://github.com/Qv2ray/Qv2ray"
+    name="Qv2ray"
+    :platforms="['linux','win','mac']"
+    description="[Discontinued] Cross-platform V2Ray client supporting Linux, Windows, macOS. Can support SSR / Trojan / Trojan-Go / NaiveProxy and other protocols through plugin system"
 />
 
 <Tool
-    url="https://github.com/2dust/v2rayN"
-    name="V2rayN"
-    :platforms="['win']"
-    description="V2RayN is a V2Ray client running on Windows."
+    url="https://github.com/MatsuriDayo/nekoray"
+    name="nekoray"
+    :platforms="['linux','win','mac']"
+    description="[Actively Updated] Supports SOCKS (4/4a/5), HTTP(S), Shadowsocks, VMess, VLESS, Trojan, TUIC, NaïveProxy (Custom Core), Hysteria2 (Custom Core or sing-box), Custom Outbound, Custom Config, Custom Core, v2ray Core"
+/>
+
+<Tool
+    url="https://github.com/v2rayA/v2rayA"
+    name="v2rayA"
+    :platforms="['linux','win','mac']"
+    description="[Actively Updated] Web GUI based cross-platform V2Ray client, supports global transparent proxy on Linux and system proxy on other platforms."
 />
 
 <Tool
-    url="https://github.com/yanue/V2rayU"
-    name="V2rayU"
+    url="https://github.com/yanue/V2rayU/tree/master"

```

Comment:
I’m not sure if V2RayU is still actively being updated. It appears to have been deleted.